spec/lib/open_food_network/user_balance_calculator_spec.rb

```diff
@@ -76,7 +76,7 @@ module OpenFoodNetwork
         }
 
         it "does not include canceled orders in the balance" do
-          expect(UserBalanceCalculator.new(o4.email, hub1).balance).to eq(-9) # = 15 + 2 - 13 - 13
+          expect(UserBalanceCalculator.new(o4.email, hub1).balance).to eq(11) # = 15 + 2 + 20 - 13 - 13
```

Comment:
There's lots of single-letter variables in this spec like `p4`, `o3`. Can we use more descriptive naming like `order_cancelled` instead of `o4`? Or at least `order4`?

Reply:
There's also a lot of **magic numbers**. The `10 + 3` numbers in the comments are default values from the `order_with_totals_and_distribution` factory, right?

Instead of the comment `# = 15 + 2 + 20 - 13 - 13`, could we make it more clear/explicit where all the numbers are coming from? Something like:
```ruby
expected_balance = payment1.amount + payment2.amount + payment4.amount - order1.total - order2.total
expect(UserBalanceCalculator.new(user1.email, hub1).balance).to eq(expected_balance)
```

Maybe we could name `p4` as `cancelled_order_payment` for extra clarity?

Reply:
Agree on the single letter variables. The magic numbers are only in comments. They could just as easily be deleted and no one would notice.

Reply:
now that I re-review I see the name of this test should be better, it says it does not include cancelled orders but it includes payments in cancelled orders.
it should be more specific " it "includes complete payments in canceled orders but does not include the canceled orders amount" do"